src/VocabularyManager.Core/Services/WordListService.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Ardalis.Specification;
+using VocabularyManager.Core.Entities;
+using VocabularyManager.Core.Exceptions;
+using VocabularyManager.Core.Interfaces;
+using VocabularyManager.Core.Specifications;
+
+namespace VocabularyManager.Core.Services
+{
+    public class WordListService : IWordListService
+    {
+        private readonly IRepositoryBase<WordList> _wordListRepository;
```

Comment:
I don't see interface of `IRepositoryBase` nowhere in this PR. Does this code compiles?

Reply:
That's from ardalis' package [specifications](https://github.com/ardalis/Specification)

Reply:
didn't knew that one. 

Something for you to read about misusage of repositories
https://medium.com/@opflucker/repository-pattern-common-implementation-mistakes-69ae95b63d3c